color_range_list:
  black:
    max:
    - 136
    - 255
    - 56
    min:
    - 0
    - 0
    - 0
  blue:
    min:
    - 90
    - 65
    - 57
    max:
    - 118
    - 255
    - 255
  green:
    max:
    - 85
    - 255
    - 255
    min:
    - 60
    - 55
    - 65
  red:
    max:
    - 180
    - 255
    - 255
    min:
    - 163
    - 138
    - 114
  white:
    max:
    - 180
    - 35
    - 255
    min:
    - 0
    - 0
    - 145